docs/serving/autoscaling/scale-bounds.md

```diff
@@ -157,18 +157,14 @@ When the Revision is created, the larger of initial scale and lower bound is aut
           allow-zero-initial-scale: "true"
     ```
 
-
-
-
-
 ## Scale Down Delay
 
 Scale Down Delay specifies a time window which must pass at reduced concurrency
 before a scale-down decision is applied. This can be useful, for example, to
 keep containers around for a configurable duration to avoid a cold start
 penalty if new requests come in. Unlike setting a lower bound, the revision
 will eventually be scaled down if reduced concurrency is maintained for the
-delay period.
+delay period. *only supported for KPA*
```

Comment:
Maybe convert this to something like:

`!!! note "Only Supported for the default KPA autoscaler class"`

(so we get nice formatting)